scripts/prebuild.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
```

Comment:
I would avoid using js for anything. Consider making this in TS and using ts-node to run it?